quest_test/test_alias.py

```diff
@@ -0,0 +1,106 @@
+import asyncio
+import pytest
+import logging
+
+from quest import WorkflowManager
+from quest.persistence import InMemoryBlobStorage, PersistentHistory
+from src.quest import step, queue
+from src.quest.historian import Historian
+from src.quest.wrappers import task
+from src.quest.manager_wrappers import alias
+
+# TODO: test exception on alias dict collision
+
+@pytest.mark.asyncio
+async def test_alias():
+    pause = asyncio.Event()
+    data_a = []
+    data_b = []
+
+    async def workflow_a():
```

Comment:
Not sure how the resources work, if I should access the queue within the alias or outside.

Reply:
Open the queue first. That way it will stay open for longer than the alias is in scope.

Reply:
```py
async with queue('data', None) as q:
    async with alias('the_foo'):
        data_a.append(await q.get())
    await pause.wait()
    data_a.append(await q.get())
```